lib/app/widget/input/selection.dart

```diff
@@ -23,7 +23,19 @@ class SelectionWidget extends StatefulWidget {
 }
 
 class SelectionWidgetState extends AppBaseWidgetState<SelectionWidget> {
-  String? selectedValue; // 현재 선택된 값
+  String? _selectedValue; // 현재 선택된 값
+
+  @override
+  void initState() {
+    super.initState();
+
+    // 예외 처리: options가 비어있으면 리턴
+    if (widget.options.isEmpty) {
+      return;
+    }
+
+    _selectedValue = widget.options.first;
```

Comment:
해당 코드를 수정해주신 덕분에 아래와 같이 좀 더 간결한 형태로 수정할 수 있을 것 같습니다.
```suggestion
    _selectedValue = widget.options.firstOrNull;
```

Reply:
제 생각에도 이 부분은 생각보다 좋은 것 같습니다.
혹시 금빈님만 괜찮으시다면 위 부분으로 수정하시고 머지를 진행해도 괜찮을까요?

Reply:
오 firstOrNull 평소에 잘 안 썼더니 생각도 못하고 있었네요! 수정하도록 하겠습니다!